zio-http/src/main/scala/zio/http/model/headers/values/Origin.scala

```diff
@@ -26,26 +28,14 @@ object Origin {
         }
       case InvalidOriginValue                   => ""
     }
-
-  def toOrigin(value: String): Origin = if (value == "null") {
```

Comment:
This implementation doesn't support origins without a port. Replacing with zio.http.URL decoder